charts/drax/charts/e2-t/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: e2-t
 description: The E2 Termination
 type: application
 version: 2.0.0
-appVersion: "e2-5.2.16"
+appVersion: "e2-6.0.0"
```

Comment:
```suggestion
# renovate: image=accelleran/e2smrcappl versioning=semver
appVersion: "6.0.0"
```

Reply:
This should also be indicated as a breaking change in the commit message (already did it for the PR title)